public/desktop/index.html

```diff
@@ -13,13 +13,34 @@
     <link rel="stylesheet" href="../css/draggable.css">
 
   </head>
-  <body>
-
+  <body id="desktop"> 
+    <div class="login">
```

Comment:
what do u think about adding the `draggable` class to this as well? 
```html
    <div class="login draggable">
```
this would make the login window draggable && feel more like a "desktop". if so u will also need to make a couple of edits to ur `.login` rule for this to work properly. specifically u'll need to remove the `margin: 10% auto;` && center it using left/top properties instead, something like:
```css
.login {
  width: 284px;
  height: 354px;
  /* ...other styles omitted for brevity...  */
  left: calc(50vw - 142px);
  top: calc(50vh - 177px);
}
```
in order to use the top/left properties we also need to specify a `position` value, but that property is being specified in the `.draggable` class, so if u add draggable to this div it will essentially get all the styles in both login && draggable applied to it, so we don't need to include a `position` value in here b/c it's already in draggable. i'm using the CSS `calc()` function to calculate the "center" position, by placing the div (relative to the top left corner) at the center of the page (50vw, 50vh) && then subtracting half of it's width/height so that it appears relative to the center of the div (rather than relative to the top/left corner of the div)

as i was testing all this out, i noticed that there were some issues w/the draggable code i originally wrote which u'll need to fix for me (sorry ^__^) in order for everything to work propertly:

first, in the `public/css/draggable.css` file remove the `left: 0px;` && the `right: 0px;` from the `.draggable` class

then, in the `public/js/utils/draggable.js` change lines 9 && 10
```js
  const left = parseInt(ele.style.left) || 0
  const top = parseInt(ele.style.top) || 0  
```
to this:
```js
  const left = parseInt(ele.style.left) || ele.offsetLeft
  const top = parseInt(ele.style.top) || ele.offsetTop
```